### PR TITLE
Implementing AccessibilityInfo.DisplayOptions ex: isInvertColorsEnabled

### PR DESCRIPTION
**Pull request by graytmatterMS**
#### Please select one of the following
- [ ] I am removing an existing difference between facebook/react-native and microsoft/react-native-macos :thumbsup:
- [ ] I am cherry-picking a change from Facebook's react-native into microsoft/react-native-macos :thumbsup:
- [X] I am making a fix / change for the macOS implementation of react-native
- [ ] I am making a change required for Microsoft usage of react-native

## Summary

When the MacOS fork was made all the AccessibilityInfo functionality was stubbed out because the iOS Accessibility APIs are different from the MacOS APIs. I am implementing several AccessibilityInfo properties and EvenListeners. This will allow a application to tell if several Accessible Display Options are enabled, or if they change.

Specifically this PR implements AccessibilityInfo.isInvertColorsEnabled AccessibilityInfo.isReduceMotionEnabled and AccessibilityInfo.isReduceTransparencyEnabled.

## Changelog

[MacOS] [Added] - Implemented AccessibilityInfo.isInvertColorsEnabled, isReduceMotionEnabled, & isReduceTransparencyEnabled

## Test Plan
1. Run RNTester
2. Go to the accessibility page 
3. Scroll to the Display Options Enabled section (second to last)
4. Open Settings>Accessibility>Display
5. Toggle Invert Colors, Reduce Motion, and Reduce Transparency.
- You should see the text change back and forth between for example "Invert colors iss disabled" and "Invert colors is enabled"

Screen Shot 2020-06-16 at 14 51 12

Screen Shot 2020-06-16 at 14 50 41

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-macos/pull/455)

### CODE REVIEW

File: RNTester/js/examples/Accessibility/AccessibilityExample.js [hunk 1]
```diff
@@ -512,17 +512,47 @@ class AccessibilityActionsExample extends React.Component {
     );
   }
 }
-class ScreenReaderStatusExample extends React.Component<{}> {
+class DisplayOptionsStatusExample extends React.Component<{}> {
```
Comment (tom-un) [inline RNTester/js/examples/Accessibility/AccessibilityExample.js:515]: Enhancing the RNTester examples to exercise the features is awesome.   But we need to 1) contribute the change back upstream, and 2) mark the deltas from upstream with the `// [TODO` , `// ]TODO` comment blocks.    In this case, as they are changes that work equally well for iOS and should be contributed upstream, the comments can be `// [TODO(OSS Candidate ISS#2710739)` and `// ]TODO(OSS Candidate ISS#2710739)`.   You can do a diff of this file against the version in fb/master.

File: RNTester/js/examples/Accessibility/AccessibilityExample.js [hunk 2]
```diff
@@ -512,17 +512,47 @@ class AccessibilityActionsExample extends React.Component {
...
+    AccessibilityInfo.addEventListener(
+      'reduceTransparencyChanged',
+      this._handleReduceTransparencyToggled,
+    );
+    AccessibilityInfo.isReduceTransparencyEnabled().done(isEnabled => {
+      this.setState({
+        reduceTransparencyEnabled: isEnabled,
+      });
+    });
+
```
[the remark this answers is not in the document]
Comment (tom-un) [reply RNTester/js/examples/Accessibility/AccessibilityExample.js:550]: just run `yarn lint --fix`

File: React/Modules/MacOS/RCTAccessibilityManager.m [hunk 3]
```diff
@@ -47,6 +55,24 @@ - (void)dealloc
                                         context:AccessibilityVoiceOverChangeContext];
 }
```
Comment (tom-un) [inline React/Modules/MacOS/RCTAccessibilityManager.m:56]: Need to stop observing the notification here.

**Comment by tom-un** [comment]
Uh, oh.  The macOS CI tests crashed in the AccessiblityExample LoadAllPages tests.    https://dev.azure.com/ms/react-native/_build/results?buildId=88462&view=logs&j=628dfcdf-afa5-5702-e4da-dfa310a93062&t=65c699ef-104f-5f6e-4efa-350cb1662389.

You can run the test locally: Cmd+U to run all the test, then you can manually re-run the LoadAllPages tests by clicking the little run arrow on that test.   The only guess I have as to why its crashing is because you're not unobserving the notification in the dealloc.

File: React/Modules/MacOS/RCTAccessibilityManager.m (hunk 3 above)
Comment (graytmatterMS) [reply React/Modules/MacOS/RCTAccessibilityManager.m:57]: I think this should successfully stop observing.

**Pull request merged**